src/GitHub.VisualStudio/UI/Views/PullRequestListView.xaml.cs

```diff
@@ -59,8 +59,10 @@ public PullRequestListView(IGitHubServiceProvider serviceProvider)
 
         bool disposed;
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA1816:CallGCSuppressFinalizeCorrectly")]
-        public void Dispose()
+        protected override void Dispose(bool disposing)
         {
+            base.Dispose(disposing);
+
             if (disposed) return;
```

Comment:
For the pattern to be correctly implemented, this whole block should be inside `if (disposing)`